core/service/src/components.rs

```diff
@@ -345,7 +345,7 @@ pub trait Components: Sized + 'static {
 	type RuntimeApi: Send + Sync;
 	/// A type that can start the RPC.
 	type RPC: StartRPC<Self>;
-	// TODO [ToDr] Traitify transaction pool and allow people to implement their own. (#1242)
+	// FIXME [ToDr] Traitify transaction pool and allow people to implement their own. (#1242)
```

Comment:
```suggestion
	// TODO: Traitify transaction pool and allow people to implement their own. (#1242)
```

switch back to ToDo, not waiting for other code.